src/content/Angular-CSharp/ClientApp/src/styles.css

```diff
@@ -5,6 +5,10 @@ a {
   color: #0366d6;
 }
 
+.btn:focus, .btn:active:focus {
+  box-shadow: 0 0 0 0.1rem white, 0 0 0 0.25rem #258cfb;
+}
```

Comment:
Should we apply the same styling that was used to fix a similar issue in IdentityUI? https://github.com/dotnet/aspnetcore/pull/42840

That way there won't be slightly-differently-styled buttons in the same app when using individual auth.

(Same question applies for the React template)

Reply:
I agree about making these consistent, but is the styling in #42840 actually used? In an ongoing email thread, @HaoK is saying that file doesn't even get loaded by default. If that's the case, does #42840 really fix/change anything?

@HaoK, could you comment on how #42840 works if the file isn't loaded?

Reply:
Yeah that's why that email thread exists, the accessibility vendor wasn't able to verify the fix for 42840 in the templates, the fix worked in the identity samples that I used to verify the css changes did what I wanted, which is how I discovered that the templates don't actually use identity's site.css by default...

Reply:
Understood, thanks for clarifying.